go/libkb/util_windows.go

```diff
@@ -183,6 +238,10 @@ func RemoteSettingsRepairman(g *GlobalContext) error {
 	oldConfig := filepath.Join(oldHome, configName)
 	if oldExists, _ := FileExists(oldConfig); oldExists {
 		if currentExists, _ := FileExists(currentConfig); !currentExists {
+			g.Log.Info("RemoteSettingsRepairman moving from %s to %s", oldHome, currentHome)
```

Comment:
I agree that it is tricky, but If the new directory has config.json and not secretkeys, then the user hasn't reprovisioned and moving secretkeys file would fix the situation.  If secretkeys and config.json exist in the new directory, then it shouldn't try to repair.